history/taskana-simplehistory-provider/src/test/java/acceptance/events/task/CreateHistoryEventOnTaskTransferAccTest.java

```diff
@@ -70,40 +82,48 @@ Stream<DynamicTest> should_CreateTransferredHistoryEvent_When_TaskIsTransferred(
           assertThat(events).hasSize(1);
           assertTransferHistoryEvent(
               events.get(0).getId(),
-              "WBI:100000000000000000000000000000000001",
+              q.getThird(),
               "WBI:100000000000000000000000000000000006",
               "admin");
         };
 
-    return DynamicTest.stream(testCases.iterator(), Triplet::getLeft, test);
+    return DynamicTest.stream(testCases.iterator(), Quadruple::getFirst, test);
   }
 
   @WithAccessId(user = "admin")
   @TestFactory
   Stream<DynamicTest> should_CreateTransferredHistoryEvents_When_TaskBulkTransfer() {
-    List<Triplet<String, List<String>, Consumer<List<String>>>> testCases =
+    List<Triplet<String, Map<String, String>, Consumer<List<String>>>> testCases =
```

Comment:
Again, why not assign the requested tasks to WBI:100000000000000000000000000000000001 to reduce test complexity?

Reply:
as disscussed a comment why the Workbaskets are parameterized is enough.

Reply:
Ok :)